packages/snaps-cli/src/cmds/build/bundleUtils.ts

```diff
@@ -128,3 +129,50 @@ export function postProcess(
 
   return processedString;
 }
+
+/**
+ * Processes dependencies and updates argv with an options object
+ * @param argv
+ */
+export function processDependencies(argv: YargsArgs) {
+  const { depsToTranspile, transpilationMode } = argv;
+  const babelifyOptions: Record<string, any> = {};
+  if (transpilationMode === TranspilationModes.localAndDeps) {
+    const regexpStr = getDependencyRegExp(depsToTranspile as string[]);
+    if (regexpStr !== null) {
+      babelifyOptions.ignore = [regexpStr];
+    }
+  }
+  return babelifyOptions;
+}
+
+/**
+ * Processes a string of space delimited dependencies into one regex string
+ * @param dependencies
+ * @returns a regexp string
+ */
+export function getDependencyRegExp(dependencies: string[]): RegExp | null {
+  let regexp: string | null = null;
+  if (!dependencies || dependencies.includes('.') || !dependencies.length) {
+    return /\/node_modules\/(?!.+)/u;
+  }
```

Comment:
I'm thinking we should just return `null` in this case, since it will be equivalent to the RegExp you're returning. That will probably be more performant as well, since Babel won't have to call `RegExp.test()` or whatever it does with this RegExp.

Reply:
I'm suggesting that we should return `null` and not set the ignore option at all in this case!

Reply:
oooooo yeah! makes sense